browser/src/Services/BufferUpdates.ts

```diff
@@ -52,5 +52,15 @@ export class BufferUpdates {
                 this._pluginManager.notifyBufferUpdate(args, bufferLines)
             }
         })
+        
+        this._neovimInstance.on("buffer-update-incremental", (args: Oni.EventContext, bufferLine: string, lineNumber: number) => {
+            // const lastLine = args.line
+            // this._lastArgs = args
+            // this._lastBufferLines = bufferLine
+            // this._lastBufferVersion = args.version
+
+            // If we can send incremental updates, and the line hasn't changed, just send the incremental change
```

Comment:
I think here, we'd want to update `_lastBufferLines`, for sure - since that is sent over sometimes to the plugin. Calling `notifyBufferUpdateIncremental` is important because that's how we update the language services with the current line, so they can give us up-to-date completion. We'd also want to update the `_lastBufferVersion` since there will be a new change tick now.